Seohee/Lv2/124 나라의 숫자.js

```diff
@@ -0,0 +1,10 @@
+function solution(n) {
+    var answer = '';
+    let number124 = [4, 1, 2];
+
+    while (n) {
+        answer = number124[n % 3] + answer;
+        n = (n % 3 == 0) ? Math.floor(n / 3) - 1 : Math.floor(n / 3);
```

Comment:
```suggestion
        n = (n % 3 === 0) ? Math.floor(n / 3) - 1 : Math.floor(n / 3);
```
n % 3의 연산값은 number타입을 반환하기 때문에 ===을 써주시는게 좋을거 같아요.

Reply:
저랑 푼 방식이 거의 비슷하시네요!ㅎㅎ 여기 삼항연산자로 처리해주신 게 더 가독성이 좋은 것 같네요👍